Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workspace/TextFile.java

```diff
@@ -0,0 +1,63 @@
+package edu.asu.spring.quadriga.domain.impl.workspace;
+
+import org.springframework.stereotype.Component;
+
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+
+@Component
+public class TextFile implements ITextFile {
```

Comment:
why is that a component? It's just a simple domain class, doesn't need to be a component.